### ISSUE TITLE
How should the CityPersons results in the Elephant in the Room paper be reproduced?

### ISSUE THREAD

**Issue by mark-arm** [hasanirtiza/Pedestron#51]
I notice that this repository trains CityPersons for [240 epochs by default](https://github.com/hasanirtiza/Pedestron/blob/master/configs/elephant/cityperson/cascade_hrnet.py#L281). Private communication with one of the authors on the topic of reproducing CityPersons results stated that:

> we used the best checkpoint inside an interval (from epochs 10-20 for table 4 and 7), to report results on the validation set

Is configs/elephant/cityperson/cascade_hrnet.py incorrect in training for 240 epochs if the best checkpoints are taken from epochs 10-20? What setting should we use here to train and select a checkpoint that reproduces the results for CityPersons in table 4 and 7?

**Comment by hasanirtiza** [comment]
You should use the same settings as in the repo and can stop after 20 epochs if you want (you should get similar results or in the same ball park as in the paper).

**Comment by mark-arm** [comment]
Thanks, I'll do that.
